frontend/components/Pagination.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Pagination from "@material-ui/lab/Pagination";
+import PaginationItem from "@material-ui/lab/PaginationItem";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    marginBottom: theme.spacing(4),
+    padding: "2px 2px",
+    display: "flex",
+    justifyContent: "center",
+    "& .MuiPaginationItem-root": {
+      fontFamily: "Montserrat, sans-serif",
+    },
+    "& .Mui-selected": {
+      backgroundColor: "#e85f5c",
+      color: "white",
+    },
+    "@media(prefers-color-scheme:dark)":{
+      "& .MuiPaginationItem-root": {
+        color: "white",
+      }
+    },
+  },
+  pagItem: {
+    color: "#676767",
+    "&:hover:not(.selected)": {
+      backgroundColor: "#feb14b",
+      color: "white",
+    },
+    "@media(prefers-color-scheme:dark)":{
+      color: "white",
+    }
+  },
+}));
+
+const PaginationComp = ({ totalPages, paginate, page, size }) => {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
+      <Pagination
+        className={classes.root}
+        renderItem={(item) => (
+          <PaginationItem {...item} className={classes.pagItem} />
+        )}
```

Comment:
Fix the pagination bug where it loads the wrong pagination numbers when clicking the 2021 section of the timeline after 2022. 
![image](https://user-images.githubusercontent.com/80564580/173496981-b4bd42af-855e-4d3d-9d1c-fbaef5f54574.png)